NOTICE

```diff
@@ -13,10 +14,33 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-Most libraries in the following directories made by other people and
-organizations and licensed in different ways:
-* `3party`
-* `tools`
-Please refer to their LICENCE, COPYING or NOTICE files for terms of use.
-Some icons files may be copyrighted by (C) 2020 My.com B.V. (Mail.Ru Group)
-See also the `.reuse/dep5` and `data/copyright.html` file for a full list of copyright notices.
+Third-party dependencies are governed by separate licenses. Please see
+the `.reuse/dep5`, `data/copyright.html`, and individual `LICENSE`,
+`COPYING`, or `NOTICE` files for more details.
+
+Map data is from OpenStreetMap, licensed under the Open Data Commons Open
+Database License (ODbL) 1.0. You may obtain a copy of the Open Database
+License (ODbL) 1.0 at
+
+    https://opendatacommons.org/licenses/odbl/1-0/
+
+Certain project resources, including but not limited to domain names,
+trademarks, hosting accounts, payment accounts, and others, are overseen
+and managed by Organic Maps OÜ. The governance of these digital assets is
+subject to policies established by Organic Maps OÜ, in compliance with
+applicable statutory laws. Donations are collected by Organic Maps OÜ.
+
+Organic Maps OÜ is a legal entity established on 2021-05-01 under the laws
+of the Republic of Estonia and the European Union, with registration number
+16225385. The primary purpose of the entity is to shield the project's
```

Comment:
My understanding is that one of important primary purposes of the company (like basically all other LLC-type companies) is to make profit / increase company's valuation.
IMO this is very important information for the community.

Reply:
OK, I've reverted back LEGAL file.